packit/config/common_package_config.py

```diff
@@ -466,3 +466,25 @@ def __setattr__(self, name, value):
                 f"It is ambiguous to set {name}: "
                 "there is more than one package in the config."
             )
+
+    def get_package_names_as_env(self) -> dict[str, str]:
+        """Creates a dict with package_name,
+        downstream_package_name and upstream_package_name.
+
+        If the config contains multiple packages
+        raise an Exception.
+        """
+        if len(self.packages) == 1:
+            for packit_package_name, package_config in self.packages.items():
+                env = {}
+                env["PACKIT_PACKAGE_NAME"] = packit_package_name
```

Comment:
I am not sure if we need this one. I am worried this can cause confusion. The Copr team does not need this and people should be able to decide what actually they need. What do you (or others) think?

Reply:
I think it could be useful, since it can be different from `downstream_package_name`. That being said, it's not very clear from the variable name what it actually represents.

Reply:
I am ok with removing this. To me it just feels like to be more complete having it, I don't know if the user should be puzzled by this since it is the user himself that choose this name, but as I said I am ok with removing it.

Reply:
How about calling it e.g. `PACKIT_MONOREPO_PACKAGE_NAME`?

Reply:
or `PACKIT_CONFIG_PACKAGE_NAME`?